packages/date/README.md

```diff
@@ -129,7 +129,7 @@ _Parameters_
 
 -   _dateSettings_ `Object`: Settings, including locale data.
 
-<a name="zonedTimeToUtc" href="#zonedTimeToUtc">#</a> **zonedTimeToUtc**
+<a name="subMinutes" href="#subMinutes">#</a> **subMinutes**
```

Comment:
Is it a breaking change – One method added, and one removed?

Reply:
It was exported sort of by mistake before, but I'll restore it.

Reply:
If it's only been in one or two GB releases (9.4 and 9.5? -- as it was added by 
#25782), I think it might be okay to remove? It never made its way into WP Core anyway.

Reply:
> It was exported sort of by mistake before, but I'll restore it.

If it isn't on npm, we can still revert, I think.

Reply:
It isn't there:
https://www.npmjs.com/package/@wordpress/date#api

Do you think `subMinutes` deserve to be exposed as well? It would be great to double-check it and compare with the latest version on npm.

We are doing next npm publishing next Monday/Tuesday so there is still time to revisit.

Reply:
> We are doing next npm publishing next Monday/Tuesday so there is still time to revisit.

In a discussion with @ockham and @david-szabo97, we decided that it's best to revert date-fns changes for now, as there a plenty of issues being cause by the mismanagement of timezones, and other compatibility issues for plugin makers, such as https://github.com/WordPress/gutenberg/issues/27159. 

While this PR seems to be in good state, we think it's worth giving it a bit more room to be reviewed and tested, and send off the revert for release today, so that things aren't broken while we work on that, especially because we don't know what other regressions might show up.

Reversal PR: https://github.com/WordPress/gutenberg/pull/27550

> Do you think subMinutes deserve to be exposed as well? It would be great to double-check it and compare with the latest version on npm.

My thought on exposing `date-fns` functions is so that we don't have to explicitly import `date-fns` and instead go through `@wordpress/date`, sort of what `@wordpress/element` does with react hooks.